simtools/util/general.py

```diff
@@ -97,64 +97,64 @@ def validateConfigData(configData, parameters):
+    if config_data is None:
+        config_data = dict()
 
-    # Collecting all entries given as in configData.
-    for keyData, valueData in configData.items():
+    # Collecting all entries given as in config_data.
+    for key_data, value_data in config_data.items():
 
-        isIdentified = False
+        is_identified = False
         # Searching for the key in the parameters.
-        for parName, parInfo in parameters.items():
-            names = parInfo.get("names", [])
-            if keyData != parName and keyData.lower() not in [n.lower() for n in names]:
+        for par_name, par_info in parameters.items():
+            names = par_info.get("names", [])
+            if key_data != par_name and key_data.lower() not in [n.lower() for n in names]:
                 continue
             # Matched parameter
-            validatedValue = _validateAndConvertValue(parName, parInfo, valueData)
-            outData[parName] = validatedValue
-            isIdentified = True
+            validated_value = _validate_and_convert_value(par_name, par_info, value_data)
+            out_data[par_name] = validated_value
+            is_identified = True
 
         # Raising error for an unidentified input.
-        if not isIdentified:
-            msg = "Entry {} in configData cannot be identified.".format(keyData)
+        if not is_identified:
+            msg = "Entry {} in config_data cannot be identified.".format(key_data)
             logger.error(msg)
             raise UnableToIdentifyConfigEntry(msg)
 
     # Checking for parameters with default option.
     # If it is not given, filling it with the default value.
-    for parName, parInfo in parameters.items():
-        if parName in outData:
+    for par_name, par_info in parameters.items():
+        if par_name in out_data:
             continue
-        elif "default" in parInfo.keys() and parInfo["default"] is not None:
-            validatedValue = _validateAndConvertValue(parName, parInfo, parInfo["default"])
-            outData[parName] = validatedValue
-        elif "default" in parInfo.keys() and parInfo["default"] is None:
-            outData[parName] = None
+        elif "default" in par_info.keys() and par_info["default"] is not None:
+            validated_value = _validate_and_convert_value(par_name, par_info, par_info["default"])
+            out_data[par_name] = validated_value
+        elif "default" in par_info.keys() and par_info["default"] is None:
+            out_data[par_name] = None
         else:
             msg = (
-                "Required entry in configData {} ".format(parName)
+                "Required entry in config_data {} ".format(par_name)
                 + "was not given (there may be more)."
             )
             logger.error(msg)
             raise MissingRequiredConfigEntry(msg)
 
-    ConfigData = namedtuple("ConfigData", outData)
-    return ConfigData(**outData)
+    Config_data = namedtuple("Config_data", out_data)
```

Comment:
I believe it should be config_data as several lines above

Reply:
Actually it should be something different cause it's a new dict. I will name it `configuration_data` for a lack of a better name